report_pre_thesis/src/sections/methodology.tex

```diff
@@ -1,8 +1,246 @@
+Consequently, our pipeline is not identical to the original, but we have strived to make it as close as possible.
 
-\textit{Recreate the current model. Reproduce the results of the original papers. Investigate the limitations of the model and propose improvements.}
+This section is dedicated to describing the methodology of our pipeline, how it differs from the original, and which design choices we have made and why.
+Furthermore, we delve into the experiments we have conducted to evaluate the performance of our pipeline such that we can identify the components that contribute the most to the overall error.
 
+Section \ref{sec:shared_preprocessing} outlines the preprocessing steps common to both the PLS1-SM and ICA phases of the pipeline.
+In Section \ref{sec:methodology_pls1}, we focus on the specifics of the PLS1-SM phase, detailing its associated data preprocessing procedures.
+Section \ref{sec:methodology_ica} then details the ICA phase, including its unique data preprocessing steps.
+Lastly, Section \ref{sec:methodology_moc} presents a description of the MOC phase of the pipeline.
+
+\subsection{Shared Preprocessing}\label{sec:shared_preprocessing}
+Similar to the original pipeline, we have used the samples from the ChemCam calibration dataset.
+Part of the data preprocessing is shared between the PLS1-SM and ICA phases of the pipeline.
+However, each phase also has its own preprocessing steps, which are described in sections \ref{sec:pls1_data_preprocessing} and \ref{sec:ica_data_preprocessing} respectively.
+
+For each target, the data is split into five datasets, one for each location on the sample that was shot at by the laser.
+Each dataset contains CCS data stored in a \texttt{.csv} file, which is read into a Pandas dataframe\cite{pandas} using the \texttt{read\_csv} function.
+Because the CCS data includes several lines of header information, we skip these lines using the \texttt{skiprows} parameter.
+We then remove the first five shots from the data similarly to \citet{cleggRecalibrationMarsScience2017}.
+These first five shots are typically contaminated by dust that covers the target before being removed by the laser-produced shock waves.
+Finally, we mask the edges of the spectral regions of the data to eliminate increased noise, especially in areas where small intensity changes lead to large variations in the calibrated signal.
+The masked regions, which do not contain unique major element diagnostic peaks, are excluded to enhance the accuracy and reliability of the quantitative analysis.
+These regions are defined in \citet{cleggRecalibrationMarsScience2017} as 240.811 --- 246.635, 338.457 --- 340.797, 382.138 --- 387.859, 473.184 --- 492.427, and 849 --- 905.574 nm.
+% An illustration of these masked regions can be seen in figure \ref{fig:masked_regions}.
+An example of these masked regions for the \texttt{cadillac} target can be seen in figure \ref{fig:masked_regions}.
```

Comment:
Probably sample. That's what they usually call it. Should be fixed now. I also corrected it in this sentence in the ICA subsection, which now says sample instead of target:

```
In addition, because \citet{cleggRecalibrationMarsScience2017} does not mention how each of the five location datasets are used for each sample, we have decided to only use one for each sample.
This likely does not produce as accurate results as using all five location datasets would, since we do not get a full representation of the sample that was shot at by the laser, instead only getting a partial representation from a single location.
```